frontend/src/general/simpleTable/SimpleTable.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { TableBody, Table, TableContainer } from "@material-ui/core";
+import SimpleTableRow from "./SimpleTableRow";
+
+function zipProperties(properties) {
+  return Object.keys(properties).map((propertyName) => {
+    return [propertyName, properties[propertyName]];
+  });
+}
+
+export default function SimpleTable(props) {
+  const { properties, alignment } = props;
+
+  const propertiesArr = zipProperties(properties);
```

Comment:
I dont think we rly need to define zipProperties as a separate function, it's a bit jarring that the SimpleTable component declaration is not the first thing in the file. Try this instead:
```
const propertiesArr = Object.keys(properties).map((propertyName) => {
    return [propertyName, properties[propertyName]];
});
```

Reply:
oop ur absolutely correct ab6d955